scripts/common/rook.sh

```diff
@@ -368,6 +368,83 @@ function rook_operator_ready() {
     return 0
 }
 
+# In certain edge cases, while migrating away from Rook, we may encounter issues.
+# Specifically, after we execute a pvmigrate operation to migrate the PVCs and to migrate the Object store, the system
+# may transition to an unhealthy state. This problem appears to be connected to specific modules
+# [root@rook-ceph-operator-747c86774c-7v95s /]# ceph health detail
+# HEALTH_ERR 2 mgr modules have failed
+# MGR_MODULE_ERROR 2 mgr modules have failed
+#     Module 'dashboard' has failed: error('No socket could be created',)
+#     Module 'prometheus' has failed: error('No socket could be created',)
+# The proposed workaround ensures a smooth transition during the migration and upgrade processes, ultimately allowing
+# for the successful deletion of Rook. To this end, this PR automates the resolution process by rectifying the Rook Ceph
+# state and allowing the migration to proceed, given that Rook will be removed in the end. It's important to note that
+# this automated fix is only applied during the checks performed when we are in the process of migrating away from Rook
+# and when Rook's removal is the intended outcome.
+#
+# Note this method is a duplication of rook_is_healthy_to_upgrade which now is called in the migration process
+# ONLY when we are moving from Rook. We should not try to fix it in other circumstances
+function rook_is_healthy_to_migrate_from() {
+    log "Awaiting 2 minutes to check Rook Ceph Pod(s) are Running"
+    if ! spinner_until 300 check_for_running_pods "rook-ceph"; then
```

Comment:
Should this be `300` or `120`? Either update the log message or the timeout

Reply:
300 we have an issue in the text. Good catcher

Reply:
Fixed